prow/spyglass/lenses/buildlog/buildlog.css

```diff
@@ -41,11 +46,18 @@ td {
 tr {
     border: none;
 }
+
 .linenum {
     user-select: none;
     color: rgba(255,255,2552,0.6);
     text-align: right;
     vertical-align: top;
+    width: 50px;
```

Comment:
can we put a comment about "if you have > 7 digits logs you have a problem" /s ?